src/parser.ts

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
I've decided to add a separate Parser class for now to remain consistent.

Reply:
sure